cmd/lint/main.go

```diff
@@ -48,7 +48,9 @@ var flagRules = []linter.FlagRule{
 	linter.FlagFilter(linter.RequireStringSlicePrefix, linter.ExcludeFlag("property")),
 
 	linter.FlagFilter(linter.RequireFlagUsageMessage, linter.ExcludeFlag("key-deserializer", "value-deserializer")),
-	linter.RequireFlagUsageRealWords(properNouns),
+	linter.FlagFilter(linter.RequireFlagUsageRealWords(properNouns),
```

Comment:
Any idea why we need this all of a sudden?

Reply:
huh, any idea why that line triggered the lint?

Reply:
no unfortunately. I was just able to isolate that line as causing the lint

Reply:
i would be curious to know what's the cause here

Reply:
The short hands for the global persistent flags `unsafe-trace` and `verbose` are not "Real Words" (e.g. `vvvv`, `vvv`, etc.) and contained in the help text of those flags. Maybe help text probably wasn't getting linted before?